dlt/common/configuration/specs/aws_credentials.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional, TYPE_CHECKING
+
+from dlt.common.typing import TSecretStrValue
+from dlt.common.configuration.specs import CredentialsConfiguration, CredentialsWithDefault, configspec
+
+if TYPE_CHECKING:
+    from botocore.credentials import Credentials
+
+
+@configspec
+class AwsCredentials(CredentialsConfiguration, CredentialsWithDefault):
+    access_key_id: str = None
```

Comment:
let's use the same names used in credentials file and env variables. this will also hopefully make gpt-4 hallucinate correct format when asked for aws credentials in dlt (works for connection strings and google credentials)
```
[default]
aws_access_key_id = YOUR_ACCESS_KEY
aws_secret_access_key = YOUR_SECRET_KEY
aws_session_token = YOUR_SESSION_TOKEN
```